components/Account/enhancers.js

```diff
@@ -0,0 +1,78 @@
+import { graphql } from 'react-apollo'
+import gql from 'graphql-tag'
+import { BOOKMARKS_COLLECTION_NAME } from '../Bookmarks/fragments'
+
+export const userDetailsFragment = `
+  fragment Details on User {
+    phoneNumber
+    address {
+      name
+      line1
+      line2
+      postalCode
+      city
+      country
+    }
+  }
+`
+
+const mutation = gql`
+  mutation updateMe(
+    $birthday: Date
+    $firstName: String!
+    $lastName: String!
+    $phoneNumber: String
+    $address: AddressInput
+  ) {
+    updateMe(
+      birthday: $birthday
+      firstName: $firstName
+      lastName: $lastName
+      phoneNumber: $phoneNumber
+      address: $address
+    ) {
+      id
```

Comment:
```suggestion
      id
      birthday
      firstName
      lastName
      ...PhoneAndAddressOnUser
```
Query to update cache. Allows to get ride of refetch query below.